nlp_architect/models/transformers/base_model.py

```diff
@@ -281,6 +281,8 @@ def _train(
         max_grad_norm: float = 1.0,
         logging_steps: int = 50,
         save_steps: int = 100,
+        training_args: str = None,
```

Comment:
why do you provide more training args?  this should have been set in the model or the training script

Reply:
moved to model